src/vmm/src/devices/virtio/net/device.rs

```diff
@@ -6,11 +6,12 @@
 // found in the THIRD-PARTY file.
```

Comment:
The commit message body needs reworking here as well. Something like this maybe:

```
Implement VIRTIO_NET_F_MRG_RXBUF feature for VirtIO network device. It allows
receiving a single network frame into multiple descriptor chains. The amount of
descriptor chains (also known as heads) that were used is written into the
`virtio_net_hdr_v1` structure which is located a the beginning of the guest buffer.
```

Reply:
done